### PR TITLE
Add encoding to etree.tostring calls

### PR DESCRIPTION
**Pull request by replaceafill**
Connected to https://github.com/archivematica/Issues/issues/1589

### CODE REVIEW

File: src/MCPClient/lib/clientScripts/archivematicaCreateMETSTrim.py [hunk 1]
```diff
@@ -106,7 +106,6 @@ def getTrimDmdSec(job, baseDirectoryPath, sipUUID):
         maxDateMod,
     )
 
-    # print etree.tostring(dublincore, pretty_print = True)
```
Comment (sevein) [inline src/MCPClient/lib/clientScripts/archivematicaCreateMETSTrim.py:109]: print :laughing:

**Review by sevein — APPROVED** [review]
LGTM

File: src/MCPClient/lib/clientScripts/archivematicaCreateMETSTrim.py (hunk 1 above)
Comment (replaceafill) [reply src/MCPClient/lib/clientScripts/archivematicaCreateMETSTrim.py:109]: Old school cool!

**Comment by replaceafill** [comment]
Cherry-picked to `stable/1.13.x` in https://github.com/artefactual/archivematica/commit/fbda1a91d6dff086e7124fa1d7a3c7953d8755bb

**Pull request merged**